llvm/lib/ObjCopy/ELF/ELFObject.cpp

```diff
@@ -2206,8 +2206,15 @@ Error Object::removeSections(
   // Transfer removed sections into the Object RemovedSections container for use
   // later.
   std::move(Iter, Sections.end(), std::back_inserter(RemovedSections));
-  // Now finally get rid of them all together.
+  // Now get rid of them all together.
```

Comment:
Nit: whilst you're modifying this comment, please fix the grammar issue
```suggestion
  // Now get rid of them altogether.
```

Reply:
done